### PR TITLE
Support Tensor that requires grad

### PR DESCRIPTION
**Pull request by LinChen-Cherry**
Add support for visualizations of Tensor that requires grad.

Note that Tensor.detach() is called inside this support.

### CODE REVIEW

**Comment by JackUrb** [comment]
Hi @LinChen-Cherry - Thanks for the fix!

**Comment by JackUrb** [comment]
Confirmed that detaching a detached tensor is a no-op. Merging
